### PR TITLE
Added an ability to auth backend. Some api implementation.

### PR DESCRIPTION
**Pull request by 11irik**
По ошибке добавил в коммит 2 файла связанных с имплементацией api.

### CODE REVIEW

File: backend/index.js [hunk 1]
```diff
@@ -1,96 +1,12 @@
...
+const express = require('express');
 
-// If modifying these scopes, delete token.json.
-const SCOPES = ['https://www.googleapis.com/auth/calendar.readonly'];
-// The file token.json stores the user's access and refresh tokens, and is
-// created automatically when the authorization flow completes for the first
-// time.
-const TOKEN_PATH = 'token.json';
+const app = express();
+app.use(express.json({ extended: true }))
```
Comment (rzvdv) [inline backend/index.js:4]: According to expree.js API there is not extended property in express.json options https://expressjs.com/ru/api.html

File: backend/routes/calendar.routes.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+let calendarApi;
+
+getAuthenticatedClient().then(auth => {
+    calendarApi = new CalendarApi(auth = {auth});
+});
+
+router.get('/', async (req, res) => {
+    try {
+        calendarApi.listEvents();
```
Comment (rzvdv) [inline backend/routes/calendar.routes.js:14]: We should respond with events list

Comment (11irik) [reply backend/routes/calendar.routes.js]: Fixed.

File: backend/index.js (hunk 1 above)
Comment (11irik) [reply backend/index.js]: Fixed.

**Comment by 11irik** [comment]
Also, as I misadded some files, I added commit with some changes connected with them in this PR.

**Pull request merged**